spec/requests/api/v1/category_data_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'Category Data', type: :request do
+
+  describe 'GET /api/v1/category_data' do
```

Comment:
I'm talking more about the URL itself than the actual file (although there should be some consistency there too). There are some general rules when it comes to generating RESTful API URLs. 

In this particular case, you're returning  restaurant categories and as such:

`/api/v1/caterogies` or `/api/v1/restaurant_categories` makes more sense than `/api/v1/category_data`